docs/src/lens.md

```diff
@@ -1,149 +1,251 @@
 # Lenses and the `Lens` trait
 
-Let's say we're building a todo list application, and we are designing the widget
-that will represent a single todo item. Our data model looks like this:
+One of the key abstractions in `druid` along with `Data` is the `Lens` trait. This page explains what they are, and then how to use them. `Lens`es may seem complicated at first, but they are also very powerful, allowing you to write code that is reusable, concise, and understandable (once you understand `Lens`es themselves).
 
-```rust,noplaypen
-{{#include ../book_examples/src/lens_md.rs:todo_item}}
+## Fundamentals: Definition and Implementation
+
+Like Rust itself, lenses are one of those things that require effort up front to learn, but are very fun and effective to use once you understand them. This section represents the effort part of the equation. I promise if you stick with it you will reap the rewards.
+
```

Comment:
This feels to me like it's repeating the previous paragraph.

Reply:
agree, I'm reminded of the old rub, "I would've written less but I didn't have time". I find in this sort of thing it's easy to write too much, and one of the challenges is going back and cutting stuff out.